ivy/functional/backends/torch/experimental/linear_algebra.py

```diff
@@ -98,11 +99,11 @@ def diagflat(
 
 
 def kron(
-    a: torch.Tensor,
-    b: torch.Tensor,
-    /,
-    *,
-    out: Optional[torch.Tensor] = None,
+        a: torch.Tensor,
```

Comment:
These indentations are remaining too🙂
They're actually not needed to be there

Reply:
These changes were made when I took the pull from the master branch and were not done by me.

Reply:
I have fixed it from my side I'll then merge this PR now.
Thanks for contributing😊